code/obj/item/device/multitool.dm

```diff
@@ -7,7 +7,7 @@
 	throwforce = 5.0
 	throw_range = 15
 	throw_speed = 3
-	desc = "You can use this on airlocks or APCs to try to hack them without cutting wires."
+	desc = "An all-around electrical tool, capable of sending jolts of electricity into things and reading power levels, but most importantly, hacking airlocks and APCs without cutting too many wires."
```

Comment:
```suggestion
	desc = "An electrical multitool. It can generate small electrical pulses and read the wattage of power cables. It is most commonly used when interfacing with airlock and APC systems on the station."
```
idk electricity terms so this is hard but the current description is clunky. 
"reading wattage of power cables" is a bit more explanatory than reading power levels, which is a bit nebulous imo.
"small electrical pulses" also has the word pulse in it, which is what the multitool input in hacking menus is called.
i think putting hacking directly in the description is a bit weird imo, so i replaced it with "interfacing"